ElectronClient/app.js

```diff
@@ -1178,7 +1181,8 @@ class Application extends BaseApplication {
 		for (const itemId of ['copy', 'paste', 'cut', 'selectAll', 'bold', 'italic', 'link', 'code', 'insertDateTime', 'commandStartExternalEditing', 'showLocalSearch']) {
 			const menuItem = Menu.getApplicationMenu().getMenuItemById(`edit:${itemId}`);
 			if (!menuItem) continue;
-			menuItem.enabled = !!note && note.markup_language === MarkupToHtml.MARKUP_LANGUAGE_MARKDOWN;
+			const isHtmlNote = !!note && note.markup_language !== MarkupToHtml.MARKUP_LANGUAGE_HTML;
+			menuItem.enabled = isHtmlNote && viewer !== 'viewer';
```

Comment:
I was expecting I could merge this, but after all this time it's still not good. I don't know what to say since we've already went over this. If something's not clear about what should happen please ask.

Also please rename the "viewer" variable, it doesn't make sense. It's like having a variable called "plane" and setting it to "car".

Reply:
My bad, I thought I understood your last comment well enough, that was why I didn’t want to bother you.

But based on what you said, I could change the `isHtmlNote` to `isMarkDownNote` And turn the final statement to:

```isMarkDownNote = !!note && note.markup_language === MarkupToHtml.MARKUP_LANGUAGE_MARKDOWN```

With the above code,  when the note is a HTML note, `isMarkDownNote` will be `false`And when the note is a Markdown note, `isMarkDownNote` will be `true`.

How do you see this approach, is it okay?

Reply:
I’ll change that to `editorMode`

Reply:
> I’ll change that to editorMode

This is also wrong. Dude, you have 3 layouts: viewer, split, editor. So how would you call the mode? I'd say `layout` or `layoutMode`.

Reply:
> How do you see this approach, is it okay?

No, it's not. You don't have to rename the variable `isHtmlNote`. Anyway, I'm too tired now. Ttyl.

Reply:
Hi @Runo-saduwa 

tessus is right about the layouts and I think you've sorted that out now.

Concerning your question about `isHtmlNote`, tessus is also right, the variable you have in the code above is correctly named.  

However, as you suggest, you could instead implement a `isMarkdownNote`.  Either variation will work. Which one you choose to go with depends on how you use it thereafter, which one makes the your following code simpler to read and understand.

Reply:
Hi @mic704b . Runo has already made all the requested changes. The code above is outdated. I think the best way is to just click on `Files changed` and start a new review.

It's a bit confusing that these conversations are not automarically resolved as soon as the code does not exist anymore.

Reply:
Thanks @tessus, I'm a bit behind.  I just wanted to respond to Runo's question.  Although it wasn't resolving the issues that were being discussed on this thread, the use of `isMarkdownNote` was an OK option to consider for other reasons.